lib/fog/openstack/baremetal.rb

```diff
@@ -34,6 +34,8 @@ class OpenStack < Fog::Service
       request :list_nodes
       request :list_nodes_detailed
       request :patch_node
+      request :set_node_power_state
+      request :set_node_provision_state
```

Comment:
Thanks for the comments!  I'm happy to do the renaming, but I just wanted to check - it's not preferred to have the noun in the request name?  I had assumed that it was desired so that it was clear what the operation did; otherwise, we'll end up with requests/baremetal/set_power_state.rb, and it's not immediately clear what kind of object that operates on.